functional-tests.sh

```diff
@@ -437,7 +437,9 @@ function run_test()
     test_presigned_get_object
     test_cat_object
     test_mirror_list_objects
-    test_watch_object
+    if [ -z "$MINT_MODE" ]; then
```

Comment:
Can you also add a documentation of MINT_MODE in the beginning of this file? the purpose of MINT_MODE is not fully clear for me.